docs/supergraph-modeling/orderby-expressions.mdx

```diff
@@ -0,0 +1,312 @@
+---
+sidebar_position: 10
+sidebar_label: Orderby Expressions
+description:
+  "Explore OrderBy Expressions in Hasura, designed for sorting data. Discover
+  how to configure sorting rules for various data types and expose them through
+  your API."
+keywords:
+  - hasura documentation
+  - graphql data structure
+  - data domain mapping
+  - hasura ddn
+  - hasura data specification
+  - graphql schema
+  - orderby expressions
+toc_max_heading_level: 4
+seoFrontMatterUpdated: false
+---
+
+# OrderBy Expressions
+
+## Introduction
+
+Hasura provides powerful tools to control sorting and ordering data. OrderBy Expression types let you define customizable 
+sorting rules for a [model](/supergraph-modeling/models.mdx).
```

Comment:
I think it should be better portrayed that `OrderByExpressions` are defined for types instead of models.
```suggestion
sorting rules for a [type](/supergraph-modeling/types.mdx).
```